modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown

```diff
@@ -1083,6 +1083,18 @@ another task with the same name is being executed in a parent project.
 - [LPS-74867]: Update the [Liferay Source Formatter] dependency to version
 1.0.470.
 
+## 2.3.0 - 2017-10-04
+
+### Added
+- [LPS-74314]: Add the property `fileExtensions` to `FormatSourceTask`.
+
+### Changed
+- [LPS-74314]: Update the [Liferay Source Formatter] dependency to version
+1.0.471.
+
+### Fixed
+- [LPS-74314]: Fix passing of the `baseDir` property to Source Formatter.
```

Comment:
@Ithildir What are we passing the `baseDir` property from? The Gradle plugin?

Reply:
The `baseDir` is a property of `FormatSourceTask`, which means it's a property of the `formatSource` Gradle task and all other Gradle tasks of type `FormatSourceTask`. Then, when the task runs, the value of the `baseDir` property is passed to SF itself via command line arguments.

The Gradle plugin sets a default value of the `baseDir` property, but people can change it.

Reply:
@Ithildir does this sound OK/accurate?

> Fix the process of passing the `baseDir` property from the `FormatSourceTask` to the Source Formatter.

Reply:
Yes, it sounds very good! Thanks!!